mkosi.build

```diff
@@ -307,6 +307,10 @@ if [ -d mkosi.kernel/ ]; then
                 --enable MEMCG \
                 --enable MEMCG_SWAP \
                 --enable MEMCG_KMEM \
+                --enable IMA_ARCH_POLICY \
+                --enable DM_VERITY_VERIFY_ROOTHASH_SIG \
+                --enable DM_VERITY_VERIFY_ROOTHASH_SIG_SECONDARY_KEYRING \
+                --enable CONFIG_INTEGRITY_MACHINE_KEYRING \
```

Comment:
This should just be `INTEGRITY_MACHINE_KEYRING`

Reply:
whops - as said, 100% untested :-)